### PR TITLE
Update docs of String methods using ActiveSupport::Inflector [ci skip]

### PR DESCRIPTION
**Pull request by nard-tech**
### Summary

* Add references to methods defined in `ActiveSupport::Inflector`
* Add document of aliased method (`#camelcase`)

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 1]
```diff
@@ -30,6 +30,8 @@ class String
   #   'apple'.pluralize(2)         # => "apples"
   #   'ley'.pluralize(:es)         # => "leyes"
   #   'ley'.pluralize(1, :es)      # => "ley"
+  #
+  # See ActiveSupport::Inflector.pluralize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.pluralize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 2]
```diff
@@ -53,28 +55,34 @@ def pluralize(count = nil, locale = :en)
   #   'the blue mailmen'.singularize # => "the blue mailman"
   #   'CamelOctopi'.singularize      # => "CamelOctopus"
   #   'leyes'.singularize(:es)       # => "ley"
+  #
+  # See ActiveSupport::Inflector.singularize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.singularize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 3]
```diff
@@ -53,28 +55,34 @@ def pluralize(count = nil, locale = :en)
...
   # +constantize+ tries to find a declared constant with the name specified
   # in the string. It raises a NameError when the name is not in CamelCase
-  # or is not initialized.  See ActiveSupport::Inflector.constantize
+  # or is not initialized.
   #
   #   'Module'.constantize  # => Module
   #   'Class'.constantize   # => Class
   #   'blargle'.constantize # => NameError: wrong constant name blargle
+  #
+  # See ActiveSupport::Inflector.constantize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.constantize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 4]
```diff
@@ -53,28 +55,34 @@ def pluralize(count = nil, locale = :en)
...
   # +safe_constantize+ tries to find a declared constant with the name specified
   # in the string. It returns +nil+ when the name is not in CamelCase
-  # or is not initialized.  See ActiveSupport::Inflector.safe_constantize
+  # or is not initialized.
   #
   #   'Module'.safe_constantize  # => Module
   #   'Class'.safe_constantize   # => Class
   #   'blargle'.safe_constantize # => nil
+  #
+  # See ActiveSupport::Inflector.safe_constantize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.safe_constantize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 5]
```diff
@@ -88,6 +96,10 @@ def safe_constantize
   #   'active_record'.camelize(:lower)        # => "activeRecord"
   #   'active_record/errors'.camelize         # => "ActiveRecord::Errors"
   #   'active_record/errors'.camelize(:lower) # => "activeRecord::Errors"
+  #
+  # +camelize+ is also aliased as +camelcase+.
+  #
+  # See ActiveSupport::Inflector.camelize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.camelize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 6]
```diff
@@ -108,11 +120,13 @@ def camelize(first_letter = :upper)
...
   #
-  # +titleize+ is also aliased as +titlecase+.
-  #
   #   'man from the boondocks'.titleize                       # => "Man From The Boondocks"
   #   'x-men: the last stand'.titleize                        # => "X Men: The Last Stand"
   #   'string_ending_with_id'.titleize(keep_id_suffix: true)  # => "String Ending With Id"
+  #
+  # +titleize+ is also aliased as +titlecase+.
+  #
+  # See ActiveSupport::Inflector.titleize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.titleize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 7]
```diff
@@ -124,13 +138,17 @@ def titleize(keep_id_suffix: false)
   #
   #   'ActiveModel'.underscore         # => "active_model"
   #   'ActiveModel::Errors'.underscore # => "active_model/errors"
+  #
+  # See ActiveSupport::Inflector.underscore
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.underscore.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 8]
```diff
@@ -124,13 +138,17 @@ def titleize(keep_id_suffix: false)
...
   def underscore
     ActiveSupport::Inflector.underscore(self)
   end
 
   # Replaces underscores with dashes in the string.
   #
   #   'puni_puni'.dasherize # => "puni-puni"
+  #
+  # See ActiveSupport::Inflector.dasherize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.dasherize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 9]
```diff
@@ -142,6 +160,8 @@ def dasherize
   #   '::Inflections'.demodulize                         # => "Inflections"
   #   ''.demodulize                                      # => ''
   #
+  # See ActiveSupport::Inflector.demodulize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.demodulize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 10]
```diff
@@ -155,6 +175,8 @@ def demodulize
   #   '::String'.deconstantize    # => ""
   #   ''.deconstantize            # => ""
   #
+  # See ActiveSupport::Inflector.deconstantize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.deconstantize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 11]
```diff
@@ -192,6 +214,8 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-Donald-E-Knuth">Donald E. Knuth</a>
+  #
+  # See ActiveSupport::Inflector.parameterize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.parameterize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 12]
```diff
@@ -202,6 +226,8 @@ def parameterize(separator: "-", preserve_case: false, locale: nil)
   #   'RawScaledScorer'.tableize # => "raw_scaled_scorers"
   #   'ham_and_egg'.tableize     # => "ham_and_eggs"
   #   'fancyCategory'.tableize   # => "fancy_categories"
+  #
+  # See ActiveSupport::Inflector.tableize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.tableize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 13]
```diff
@@ -212,6 +238,8 @@ def tableize
   #
   #   'ham_and_eggs'.classify # => "HamAndEgg"
   #   'posts'.classify        # => "Post"
+  #
+  # See ActiveSupport::Inflector.classify
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.classify.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 14]
```diff
@@ -233,6 +261,8 @@ def classify
   #   'author_id'.humanize(capitalize: false)       # => "author"
   #   '_id'.humanize                                # => "Id"
   #   'author_id'.humanize(keep_id_suffix: true)    # => "Author Id"
+  #
+  # See ActiveSupport::Inflector.humanize
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.humanize.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 15]
```diff
@@ -242,6 +272,8 @@ def humanize(capitalize: true, keep_id_suffix: false)
   #   'what a Lovely Day'.upcase_first # => "What a Lovely Day"
   #   'w'.upcase_first                 # => "W"
   #   ''.upcase_first                  # => ""
+  #
+  # See ActiveSupport::Inflector.upcase_first
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.upcase_first.
```

File: activesupport/lib/active_support/core_ext/string/inflections.rb [hunk 16]
```diff
@@ -253,6 +285,8 @@ def upcase_first
   #   'Message'.foreign_key        # => "message_id"
   #   'Message'.foreign_key(false) # => "messageid"
   #   'Admin::Post'.foreign_key    # => "post_id"
+  #
+  # See ActiveSupport::Inflector.foreign_key
```
Comment (kamipo) [inline activesupport/lib/active_support/core_ext/string/inflections.rb]:
```suggestion
  # See ActiveSupport::Inflector.foreign_key.
```

**Comment by kamipo** [comment]
Also, you can add `[ci skip]` to your commit message for documentation-only changes to skip the CI build.

**Comment by nard-tech** [comment]
@kamipo I added `[ci skip]` to the commit message. thank you.

**Comment by nard-tech** [comment]
@kamipo I applied your suggestions. Thank you.

**Pull request merged**